Makefile

```diff
@@ -459,12 +460,10 @@ podman-remote-%-release:
 	rm -f release.txt
 	$(MAKE) podman-remote-release-$*.zip
 
-BINDINGS_SOURCE = $(wildcard pkg/bindings/**/types.go)
-.generate-bindings: $(BINDINGS_SOURCE)
+generate-bindings:
```

Comment:
I think you need `.PHONY: generage-bindings` right above this

Reply:
Thanks for checking!